packages/crayons-i18n/i18n/en-US.json

```diff
@@ -190,6 +190,34 @@
       "maxCharsWarning": "You have reached a maximum limit of {{count}} characters.",
       "maxChoiceOptionsWarning": "You have added maximum limit of {{count}} choices.",
       "maxCharsChoicesWarning": "Choice name should not exceed {{count}} characters."
+    },
+    "sections": {
+      "add": "Add Section",
+      "addTooltipTitle": "Dynamic Sections",
+      "addTooltipDescription": "Sections are conditional fields that show up based on the rules you have defined.",
+      "sectionCreatedForAllChoices": "The dropdown property is not available for adding new sections",
+      "editTooltipText": "Edit Section",
+      "deleteTooltipText":"Delete Section",
+      "hide": "Hide sections",
+      "show": "Show sections",
+      "new": "New section",
+      "sectionDescription": "Sections are conditional fields that show up based on the rules you have defined.",
+      "save": "Save",
+      "cancel": "Cancel",
+      "sectionNameEmpty": "Section name can't be empty",
+      "emptySection": "Drag and drop fields to add to this section",
+      "sectionHeading": "<span class='highlight'>{{sectionName}}</span> | Show section if <span class='highlight'>{{fieldLabel}}</span> is : <span class='highlight'>{{sectionChoiceValue}}</span>",
```

Comment:
CSS class and html tags in translation files are not recommended, try to restructure the text to avoid this

Reply:
Done